webclient/core/repository_connector.py

```diff
@@ -395,6 +401,11 @@ def _send_request(
             response = requests.patch(url, auth=auth, headers=headers, data=data)
         extra["status_code"] = response.status_code
 
+        # TODO: Remove before merge
+        # raise FedoraUpdatedByAnotherTransactionError(
+        #    url, response.text, response.status_code, response.headers, self.transaction
+        # )
+
```

Comment:
Tady je nějaké TODO - není třeba dořešit?

Reply:
Už by to nemělo být potřeba, pro případ opravy si to zase vložím.